src/graph_ir/tests/test_low_graph_souffle_emitter.cc

```diff
@@ -0,0 +1,55 @@
+#include "low_graph_ir.h"
+#include "low_graph_souffle_emitter.h"
+
+#include <vector>
+#include <string>
+#include <iostream>
+
+using namespace std;
+
+LowGraph MakeTestGraph() {
```

Comment:
Instead of creating a function that returns a global it constructs, why not just create a number of `static const` globals? If you used a constant expression as the initializer for a `const` global, you would be freed from thinking about copys, initialization, or really any control flow in building your test graph.

Reply:
ok fixed.